mods/ra/maps/allies-05a/weapons.yaml

```diff
@@ -2,19 +2,14 @@ MissionColt:
 	ReloadDelay: 6
 	Range: 7c0
 	Report: gun5.aud
-	Projectile: Bullet
-		High: true
-		Speed: 1c682
+	Projectile: InstantHit
 	Warhead@1Dam: SpreadDamage
 		Spread: 42
 		Damage: 50
 		Versus:
-			None: 0
 			Wood: 10
-			Light: 0
-			Heavy: 0
-			Concrete: 0
 		DamageTypes: Prone50Percent, TriggerProne, BulletDeath
+		AffectsParent: true
 	Warhead@2Eff: CreateEffect
 		Explosions: piffs
 		InvalidImpactTypes: Water
```

Comment:
Well, `piffs` was correct. At least that's how I remember it from the original. (Only one piff would look even weirder.)
I'm fine with removing the piffs, but am really not convinced about removing the damage entirely. Tanya is going on a rampage after all, isn't she?